src/storage/test/AdHocSchemaManager.cpp

```diff
@@ -166,8 +166,11 @@ StatusOr<GraphSpaceID> AdHocSchemaManager::toGraphSpaceID(folly::StringPiece spa
 // This interface is disabled
 StatusOr<TagID> AdHocSchemaManager::toTagID(GraphSpaceID space, folly::StringPiece tagName) {
     UNUSED(space);
-    UNUSED(tagName);
-    LOG(FATAL) << "Unimplement";
+    try {
```

Comment:
please check the comment

Reply:
Good catch.  The comment should be fixed.  
AdHocSchemaManager is only used in UTs,  so LOG(FATAL) is ok here.